k9mail/src/main/java/com/fsck/k9/activity/compose/AttachmentPresenter.java

```diff
@@ -162,19 +165,54 @@ public boolean loadNonInlineAttachments(MessageViewInfo messageViewInfo) {
                 allPartsAvailable = false;
                 continue;
             }
-            addAttachment(attachmentViewInfo);
+            if(loadNonInlineAttachments) {
+                addAttachment(attachmentViewInfo);
+            }
         }
 
         return allPartsAvailable;
     }
 
+    public boolean loadNonInlineAttachments(MessageViewInfo messageViewInfo) {
```

Comment:
Code style generally prefers the public function above the private functions it uses.